utils/make_jsonld_context.py

```diff
@@ -68,7 +68,6 @@
     "propositional_part_of": "crm:P148i_is_component_of",
     "timespan_part_of": "crm:P86_falls_within",
     "location_part_of": "crm:P89_falls_within",
-    "interest_part_of": "la:interest_part_of"
```

Comment:
Is this unrelated?

Reply:
Yep, unrelated to the person stuff. Just noticed it was vestigial when looking at this part of the code.